core/tests/test_utils.py

```diff
@@ -483,12 +483,32 @@ def get_expected_logout_url(self, slug):
         """Returns the expected logout URL."""
         return current_user_services.create_logout_url(slug)
 
-    def get_response(self, url, params=None, expect_errors=False,
-                     expected_status_int=200,
-                     expected_content_type='text/html'):
-        """Get a response, transformed to a Python object."""
+    def get_response(
+            self, url, expected_content_type, params=None,
+            expected_status_int=200):
+        """Get a response, transformed to a Python object.
+
+        Args:
+            url: str. The URL to fetch the response.
+
+            expected_content_type: str. The content type to expect.
+
+            params: str or dict. A query string, or a dictionary that will be
+            encoded into a query string.
+
+            expected_status_int: int. The integer status code to expect. Will
+            be 200 if not specified.
+
+        Returns:
+            webtest.TestResponse. The test response.
+        """
+        have_errors = False
+        if expected_status_int >= 400 and (
+                expected_status_int < 600):
```

Comment:
Don't think you need this second part -- just expected_status_int >= 400 should suffice.

Reply:
done